packages/fluentui/react-northstar-prototypes/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@fluentui/react-northstar-prototypes",
+  "description": "A set of prototypes built using @fluentui/react-northstar",
+  "version": "0.51.0",
+  "bugs": "https://github.com/microsoft/fluentui/issues",
```

Comment:
Let's add `private: true` to avoid publishing of this package

Reply:
Okay